packages/shorebird_code_push_protocol/lib/src/models/user.dart

```diff
@@ -18,4 +18,7 @@ class User {
 
   /// The unique user identifier.
   final int id;
+
+  /// Whether the user has an active subscription.
```

Comment:
This comment doesn't add much but I'm struggling to think of a better doc.  "Whether this is a paying user" maybe?

Reply:
Yeah good call I can improve this.